scripts/metadata_parser.py

```diff
@@ -95,7 +103,12 @@ def merge_metadata_dicts(x, y):
 
 
 def parse_variable_tables(filename):
-    """DOCUMENTATION MISSING"""
+    """Parses metadata tables on the host model side that define the available variables.
+    Metadata tables can describe to variables inside a module or as part of a derived
```

Comment:
I am not sure what is meant by "describe to variables".

Reply:
That should read "can refer to variables". I will fix this right on github.